deploy/kubernetes/overlays/stable/ecr-public/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
+  - ../../base
+images:
+  - name: 602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/aws-fsx-csi-driver
```

Comment:
What does this do? (why the reference to `602401143452` here)

Reply:
That value is the account ID owning the repos with Amazon EKS-vended container images - https://docs.aws.amazon.com/eks/latest/userguide/add-ons-images.html